### DISCUSSION TITLE
请求增加一个可以拖动的步骤条

### DISCUSSION THREAD

**Discussion by yangf85** [dotnetcore/BootstrapBlazor#3202] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

希望增加一个新功能：一个可拖动的步骤进度条组件。该组件应允许用户通过一系列步骤直观地跟踪他们的进度，并且能够拖动当前步骤标记到任意步骤。此外，进度条中的每个步骤应该能够绑定到预定义枚举的特定值，从而提供更大的灵活性，并与我们现有的数据模型更好地集成。

### Describe the solution you'd like

- 步骤进度条应该支持鼠标拖动操作，用户可以通过拖动来改变当前步骤。
- 进度条应该能够与枚举类型的变量进行绑定，每个步骤对应枚举中的一个值。
- 如图
- 
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/1f8606b5-b8d3-478a-8aa5-e7d3d40f8880)

**Comment by bb-auto** [comment]
@yangf85 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
@yangf85 希望能来个 PR 帮忙实现这个功能，以目前文字描述，还无法完全理解需求
